sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3705,6 +3705,24 @@ public partial class EncryptionSettingsGroup
         public static bool operator !=(Azure.ResourceManager.Compute.Models.ExpandTypesForListVm left, Azure.ResourceManager.Compute.Models.ExpandTypesForListVm right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct FileFormat : System.IEquatable<Azure.ResourceManager.Compute.Models.FileFormat>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public FileFormat(string value) { throw null; }
+        public static Azure.ResourceManager.Compute.Models.FileFormat VHD { get { throw null; } }
```

Comment:
From the convention of dotnet abbreviations, we should have `Vhd` instead of `VHD` here.
You could do this by adding a new entry `VHD: Vhd` to the `rename-rules` section in the `autorest.md` file

Reply:
Changed VHD and VHDX to Vhd and Vhdx